### PR TITLE
Update CI scheduled jobs

### PR DESCRIPTION
**Pull request by SeanTAllen**
Most scheduled jobs have been switched to being triggered by events that
happen in other repos.

### CODE REVIEW

File: ci/triggered-jobs.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+- [corral: breakage-against-ponyc-latest](https://github.com/ponylang/corral/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [crypto: breakage-against-ponyc-latest](https://github.com/ponylang/crypto/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [glob: breakage-against-ponyc-latest](https://github.com/ponylang/glob/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [http: breakage-against-ponyc-latest](https://github.com/ponylang/http/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [http_server: breakage-against-ponyc-latest](https://github.com/ponylang/http_server/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [net_ssl: breakage-against-ponyc-latest](https://github.com/ponylang/net_ssl/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [peg: breakage-against-ponyc-latest](https://github.com/ponylang/peg/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [ponyup: breakage-against-linux-ponyc-latest](https://github.com/ponyup/appdirs/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
```
Comment (EpicEric) [inline ci/triggered-jobs.md]:
```suggestion
- [ponyup: breakage-against-linux-ponyc-latest](https://github.com/ponylang/ponyup/blob/master/.github/workflows/breakage-against-linux-ponyc-latest.yml)
```

File: ci/triggered-jobs.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- [reactive_streams: breakage-against-ponyc-latest](https://github.com/ponylang/reactive_streams/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [regex: breakage-against-ponyc-latest](https://github.com/ponylang/regex/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [rfc-tool: breakage-against-ponyc-latest](https://github.com/ponylang/rfc-tool/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [semver: breakage-against-ponyc-latest](https://github.com/ponylang/semver/blob/master/.github/workflows/breakage-against-ponyc-latest.yml)
+- [shared-docker: linux-builder-update]()
```
Comment (EpicEric) [inline ci/triggered-jobs.md]: This link is empty.

**Review by EpicEric — CHANGES_REQUESTED** [review]

File: ci/triggered-jobs.md (hunk 2 above)
Comment (SeanTAllen) [reply ci/triggered-jobs.md]: Fixed.

**Comment by SeanTAllen** [comment]
@EpicEric can you give this another review? if it's good, can you squash and merge?

**Review by EpicEric — APPROVED** [review]

**Pull request merged**